docs/docsite/rst/user_guide/intro_patterns.rst

```diff
@@ -1,85 +1,132 @@
 .. _intro_patterns:
 
-Working with Patterns
-=====================
+Patterns: targeting hosts and groups
+====================================
```

Comment:
Do end users understand 'patterns' in this context? I'm thinking - what would I search for? and for me, I'd search something like 'how to limit my ansible playbook to a set of hosts'? or something like that. I wouldn't think to call it ansible patterns for example.

Reply:
I agree it's obscure - I didn't understand what a "pattern" was at first myself. But it is the accepted term, and "limit" means something different in Ansible. If we use "pattern" elsewhere in the User Guide, I hope the term and concept will gain more traction. Expanding the title should help users find this content - it's tough to create a title that offers context without getting too long for the left nav. Alternate ideas welcome!